db/db_impl/db_impl.h

```diff
@@ -393,6 +393,9 @@ class DBImpl : public DB {
   virtual void GetLiveFilesMetaData(
       std::vector<LiveFileMetaData>* metadata) override;
 
+  virtual Status GetLiveFilesChecksumInfo(
```

Comment:
This same information is available through GetLiveFilesMetaData, right? Are we adding this function for simplicity in getting the information? Or better performance?

I also think it's worth mentioning that this function (and others) are typically called with file deletions disabled, to avoid TOCTOU bugs / race conditions in cases where the information is expected to be current even after the call has completed.

Reply:
Yes, `GetLiveFilesMetaData` gives us `MetaData` but it has more than what we need. I think using `GetLiveFilesChecksumInfo` is more efficient as it provides exactly what we want. (Thanks for pointing out these functions are typically called with file deletions disabled :)